.github/workflows/release-pr-check.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Check PR author
         id: check_author
         run: |
-          if [ "${{ github.actor }}" != "aqua-bot" ]; then
+          if [ "${{ github.event.pull_request.user.login }}" != "aqua-bot" ]; then
```

Comment:
It's better to use envs.
https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable

Reply:
yeah, right.
updated in https://github.com/aquasecurity/trivy/pull/8702/commits/c0e186bcf233f614c3ea06e800ff101356590166